# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/test/mochitest.ini
# Commit: 473ebe5ec7aa
# Full Hash: 473ebe5ec7aa4f1c5da45ed580fb56258737a210
# Author: Boris Chiou <boris.chiou@gmail.com>
# Date: 2023-08-23 04:21:33
# Regressor Bug: 1819464
# File Overlap Count: 2
# Description:
#   Bug 1819464 - Part 2: Add content-box and stroke-box into transform-box in style. r=emilio
#   
#   Also, add a prefernce for content-box and stroke-box. So we have to
#   define a new type, TransformBox, and let content-box and stroke-box
#   behind the preference.
# ==============================================================================

diff -r 9c22e16e0567 -r 473ebe5ec7aa layout/style/test/mochitest.ini
--- a/layout/style/test/mochitest.ini	Tue Aug 22 19:49:03 2023 +0000
+++ b/layout/style/test/mochitest.ini	Tue Aug 22 19:49:03 2023 +0000
@@ -25,6 +25,7 @@
   layout.css.animation-composition.enabled=true
   layout.css.basic-shape-rect.enabled=true
   layout.css.basic-shape-xywh.enabled=true
+  layout.css.transform-box-content-stroke.enabled=true
 support-files =
   animation_utils.js
   bug1729861.js